content/applications/essentials/activities.rst

```diff
@@ -71,17 +72,16 @@ List view
 
 Activities can also be created from a list view.
```

Comment:
Just to stay consistent with the rest of the doc, add the view's icon here too please :)
```suggestion
Activities can also be created from a :icon:`oi-view-list` :guilabel:`(list)` view.
```